src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -37,9 +38,11 @@
     "require-dev": {
         "phpspec/phpspec": "^2.4",
         "akeneo/phpspec-skip-example-extension": "~1.2",
-        "doctrine/mongodb-odm": "1.0.*@dev",
+        "doctrine/mongodb-odm": "^1.0",
```

Comment:
Yes and we don't have it on 7.0 yet I think, so we will need to live with a failing build anyway because excluding mongodb examples for 7.0 only is work that requires more time than I have right now. :P We can look at it shorlty after release I guess.